src/integrations/feed/types.ts

```diff
@@ -57,8 +58,16 @@ export type FeedConfig = {
 
 export type DynamicConfig = Omit<FeedConfig, 'total_pages'>;
 
+export type FeedConfigGeneratorResult = {
+  config: FeedConfig;
+  tyr_metadata?: TyrMetadata;
```

Comment:
Check my comment about making it more generic

Reply:
Yes!